### PR TITLE
Add PGP fingerprint to 020_installation.rst

### PR DESCRIPTION
**Pull request by palbr**
I like the idea of verifying the integrity of applications, I download from the internet. So I was very happy to see that restic does provide SHA256-checksums which are signed with the maintainers PGP key.

The only thing I miss: I could not find a direct way to download the used PGP key and verify the keys fingerprint.

Doing some searches, I found:
* https://github.com/restic/rest-server/issues/121
* https://restic.net/blog/2015-09-16/verifying-code-archive-integrity/

To help other restic users, I think you should add information about your PGP key/fingerprint to this installation doc, too. To save you some precious time, I created a draft, how this doc might be expanded, in this pull-request. You are free to accept it or change the text to your liking.

I copied the key/fingerprint text from: ``restic/restic/master/doc/090_participating.rst``

Thank you for your work in restic!

What does this PR change? What problem does it solve?
-----------------------------------------------------

My pull request enhances the installation documentation.

Was the change discussed in an issue or in the forum before?
------------------------------------------------------------

Only discussion I found is in the neighbor project "rest-server": https://github.com/restic/rest-server/issues/121

Checklist
---------

- [x] I have read the [Contribution Guidelines](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#providing-patches)
- [x] I have enabled [maintainer edits for this PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
- [-] I have added tests for all changes in this PR  -> no tests for "doc update" possible
- [x] I have added documentation for the changes (in the manual)
- [-] There's a new file in `changelog/unreleased/` that describes the changes for our users (template [here](https://github.com/restic/restic/blob/master/changelog/TEMPLATE))
- [-] I have run `gofmt` on the code in all commits -> not applicable
- [x] All commit messages are formatted in the same style as [the other commits in the repo](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#git-commits)
- [x] I'm done, this Pull Request is ready for review

### CODE REVIEW

**Comment by fd0** [comment]
Thanks for your contribution! I think it'd be even better to also include my GPG key as a file, so users don't have to rely on third-party keyserver infrastructure (which still seems to pretty much broken). What do you think?

**Comment by palbr** [comment]
> ... I think it'd be even better to also include my GPG key as a file, so users don't have to rely on third-party keyserver infrastructure (which still seems to pretty much broken). What do you think?

I agree: Retrieving GPG keys from public keyservers might be cumbersome / problematic sometimes. So providing your GPG key on the restic website as a file, too, will make the process of verifiing easier. If people insist to download your key from a key server, the still can do this. Your key on the restic website is just another option.

The way it is already implemented on https://restic.net/#contact seems good to me.

File: doc/020_installation.rst [hunk 1]
```diff
@@ -188,8 +188,20 @@ are considered stable and releases are made regularly in a controlled manner.
 There's both pre-compiled binaries for different platforms as well as the source
 code available for download. Just download and run the one matching your system.
 
-The official binaries can be updated in place using the ``restic self-update``
-command (needs restic 0.9.3 or later):
+On your first installation, if you desire, you can verify the integrity of your
+downloads by testing the SHA-256 checksums listed in ``SHA256SUMS`` and verifying
+the integrity of the file ``SHA256SUMS`` with the PGP signature in ``SHA256SUMS.asc``. 
+The PGP signature was created using the key (`0x91A6868BD3F7A907 <https://pgp.mit.edu/pks/lookup?op=get&search=0xCF8F18F2844575973F79D4E191A6868BD3F7A907>`__):
```
Comment (MichaelEischer) [inline doc/020_installation.rst:194]: As the key server seems to be broken, let's just use the key stored at `https://restic.net/gpg-key-alex.asc`

**Review by MichaelEischer — CHANGES_REQUESTED** [review]

File: doc/020_installation.rst (hunk 1 above)
Comment (palbr) [reply doc/020_installation.rst]: I changed the URL to the GPG keyfile. Thank you for the hint.

**Review by MichaelEischer — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**